src/routes/verify/[verifyToken]/+page.svelte

```diff
@@ -0,0 +1,90 @@
+<script>
+    // Framework imports
+    import { onMount } from "svelte";
+    import { urls, tryFetchingRestricted, toaster } from "$lib/util.js";
+    import { getToastStore, ProgressRadial } from "@skeletonlabs/skeleton";
+    import { goto } from "$app/navigation";
+
+    // Definitions
+    // Constants
+    const toastStore = getToastStore();
+
+    // Toast settings
+    const successToast = {
+        message: "Verifizierung erfolgreich",
+        bg: "primary",
+    };
+
+    const errorToast = {
+        message: "Verfizierung fehlgeschlagen",
+        bg: "error",
+    };
+
+    const serverErrorToast = {
+        message: "Verfizierung nicht möglich. Versuche es später erneut",
```

Comment:
Dann aber bitte auch vollständig korrigieren
```suggestion
        message: "Verifizierung nicht möglich. Versuche es später erneut.",
```

Reply:
Wowow, Nils findet Schreibfehler die ich übersehen habe? :exploding_head:

Reply:
Haha, hat mich auch gewundert 😆